orderbook/openapi.yml

```diff
@@ -265,6 +291,29 @@ components:
       allOf:
         - $ref: "#/components/schemas/OrderCreation"
         - $ref: "#/components/schemas/OrderMetaData"
+    Trade:
+      description: |
+        Trade data such as executed amounts, fees, order id and block number.
+      type: object
+      properties:
+        blockNumber:
+          description: "Block in which trade occurred"
+          type: integer
+        logIndex:
+          description: "Index in which transaction was included in block"
+          type: integer
+        orderUid:
+          description: "Unique ID of the order matched by this trade"
+          $ref: "#/components/schemas/UID"
+        sellAmount:
```

Comment:
Do you plan to return also the sellAmountAtomsBeforeFees like in the orders?

Reply:
Which ever you prefer... Note that `feeAmount` it already part of "Trade" so you have access all the necessary information. But I don't intend to include all three (i.e. `a,  b,  a+b`).

Reply:
It wouldn't be for me :) 
I don't need it, it's just applying the same thing we did in the orders, there was confusion if sellAmount was with or without fees, so we thought that adding the denormalized would make the API easier to use.

Personally, I don't care about this one. Just flag in it.

Reply:
The implementation at the moment returns `buy_amount`, `sell_amount` and `fee_amount`

Reply:
To make it align with the order endpoint, I would say we should return sell_amount_before_fees as well here (the redundancy was deemed ok in order to have more clarity for the consumer of the API without having to read the detailed documentation of the endpoint)

Reply:
Are you suggesting that the field `sellAmount` be renamed to `sellAmountBeforeFees` here? Or would it suffice to just include this as part of the field description? I would prefer to keep the name as small as possible if everyone is in favour.

Reply:
I would like to have both fields: sellAmount AND sellAmountBeforeFees.

Basically, the same way we have it right now in Order: https://github.com/gnosis/oba-services/blob/997a72beb01261b1d7c67b6ce40429b0b9d6e8d8/orderbook/openapi.yml#L249-L254

Reply:
Gotcha, so then I shouldn't bother with fee amount.

Reply:
Done. Check it out!

Reply:
@bh2smith why did you remove fee amount? For orders where we have:

- executedSellAmount
- executedSellAmountBeforeFees
- executedFeeAmount

The redundancy was introduced on purpose to make it easier for the consumer of this API to understand what each value implies. Could you add the fee amount back please?